### DISCUSSION TITLE
Nats docker how to pass Auth Token --auth 123

### DISCUSSION THREAD

**Discussion by khanakia** [nats-io/nats-server#4839] [Q&A]
I am trying to start Nats server using an Auth token e.g. `nats-server --auth s3cr3t`
but as you can see in below Docker Screenshot there is no option to pass Auth

https://docs.nats.io/running-a-nats-service/configuration/securing_nats/auth_intro/tokens

![Screenshot 2023-12-01 at 1 49 56 PM](https://github.com/nats-io/nats-server/assets/2174170/56d9ffed-2d7b-4226-b61a-0a0f1597a0a4)

**Comment by ripienaar** [comment]
Your container manager should allow arguments to be passed to nats server, I dont know which container manager it is, but this is a missing feature in the container manager or just there's a different way to do it.  Not a nats problem.

**Reply by khanakia** [reply]
It's Docker Desktop for Macos and i am using the latest image from here https://hub.docker.com/_/nats/

![Screenshot 2023-12-01 at 4 01 54 PM](https://github.com/nats-io/nats-server/assets/2174170/2819ebab-80d9-4395-8ba0-8c29a265ea29)

**Reply by ripienaar** [reply]
Docker desktop is not very good and just doesnt support it.  Suggest you use the CLI instead.

**Reply by khanakia** [reply]
Got it will do, Thanks.
